pkg/trace/trace.go

```diff
@@ -16,21 +16,13 @@ package trace
 
 import (
 	"fmt"
-	"os"
 	"runtime"
 	"time"
 
-	log "github.com/Sirupsen/logrus"
+	"github.com/Sirupsen/logrus"
 )
 
-var Logger = &log.Logger{
-	Out: os.Stderr,
-	// Apparently this forces terminal handling even when Out isn't a TTY (like
-	// on a VCH, where STDOUT/STDERR are currently redirected to a file).
-	Formatter: &log.TextFormatter{ForceColors: true},
-	Hooks:     make(log.LevelHooks),
-	Level:     log.InfoLevel,
-}
+var Logger = logrus.New()
```

Comment:
Yeah, this breaks our logging.  `logrus` does a syscall on the fd for STDOUT to check if it's a tty and if it is not, it blindly passes escape sequences like line-wrapping.    Yes, really.  If you want to verify, startup a VCH, and tail the port layer log and you'll see all of the `Operation` log messages will include their "\n"s and "\t"s when there is a stack trace.  Ergo the comment in the original version of this file.